### PR TITLE
chore: update wpt

### PR DESCRIPTION
**Pull request by andreubotella**
This PR also removes tests that were deleted or moved, and enables a number of WPT tests that were not listed in the expectations.

This PR does not enable the myriad of worker tests outside of the `workers` directory. That will be left to a follow-up PR.

### CODE REVIEW

File: tools/wpt/expectation.json [hunk 1]
```diff
@@ -13034,8 +13032,7 @@
     },
     "wrapKey_unwrapKey": {
       "wrapKey_unwrapKey.https.any.html": false
-    },
-    "randomUUID.any.html": true
```
Comment (lucacasonato) [inline tools/wpt/expectation.json:13038]: Where did this test go?

Comment (lucacasonato) [reply tools/wpt/expectation.json:13038]: Oh it is `randomUUID.https.any.html` now.

File: tools/wpt/expectation.json [hunk 2]
```diff
@@ -13598,6 +13649,190 @@
       "<a> and <area>.searchParams should be undefined"
     ],
     "idlharness.any.html": true,
+    "toascii.window.html": [
```
Comment (lucacasonato) [inline tools/wpt/expectation.json:13652]: These tests are never going to pass (they use DOM elements). We should remove them.

File: tools/wpt/expectation.json (hunk 1 above)
Comment (andreubotella) [reply tools/wpt/expectation.json:13038]: It was renamed to `randomUUID.https.any.js` in https://github.com/web-platform-tests/wpt/pull/28967. Looks like we added the renamed version but didn't remove the old name.

File: tools/wpt/expectation.json (hunk 2 above)
Comment (andreubotella) [reply tools/wpt/expectation.json:13652]: Some of the tests use the `<a>` and `<area>` elements, but others use the `URL` class.

**Comment by andreubotella** [comment]
`/websockets/stream/tentative/abort.any.html?wpt_flags=h2` seems to be flaky, so I've opened #12208 to track that and I'll be removing it from the expectations.

**Review by lucacasonato — APPROVED** [review]
LGTM. Thanks

**Pull request merged**